File: 05. Input-Output in Java/08_ReadCharacterValue.cpp
import java.util.Scanner;

public class ReadCharacterValue {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter a character: ");
        char a = sc.next().charAt(0);
        
        System.out.println("The Character: " + a);
    }
}

File: 05. Input-Output in Java/01. Taking Inputs/02. Using BufferedReader Class/08_ReadCharacterValue.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class _08_ReadCharacterValue {
    public static void main(String args[]) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        
        System.out.print("Enter a character: ");
        char a = br.readLine().charAt(0);
        
        System.out.println("The Character: " + a);        
    }
}